Thingvisors/DockerThingVisor/ThingVisor_LoRaWAN/README.md

```diff
@@ -0,0 +1,21 @@
+# Chirpstack LoRaWAN server ThingVisor
+
+Provide a framework to implement thingVisors connecting vThings to devices through a LoRaWAN network.
+
+## Startup and configuration
+
+The base command is like this:
+```bash
+python f4i.py add-thingvisor -c http://127.0.0.1:8090 -i fed4iot/lorawan-tv -n CSCAM01 -p "{'chirpstack_mqtt_server':'172.17.0.2','chirpstack_mqtt_port':'8883', 'chirpstack_cafile':$(base64 -w0 '/path/to/ca.crt'), 'chirpstack_crtfile':$(base64 -w0 '/path/to/user.crt'), 'chirpstack_keyfile':$(base64 -w0 '/path/to/user.key'), 'devices=[{{'type':'smartcam', 'label':'SC-ABC', 'appid':'12', 'deveui':'0123456789ABCDEF'}}]}
```

Comment:
Hi Gilles,
We are updating the way the parameters are passed to the CLI to resolve some issues. It expects the parameters argument (-p) as a json formatted string like: '{"key":"value", "key2":{"key3":"value3"}}', in particular, if the parameter is just a string: '"string"'. You can try it on the development "json-dumps-replace" branch.